addons/mail_plugin/static/src/to_translate/translations_gmail.xml

```diff
@@ -43,6 +43,7 @@
     <string>Something bad happened. Please, try again later.</string>
     <string>Invalid URL</string>
     <string>No company attached to this contact.</string>
+    <string>The attachments are ignored because the total size exceed the limit (40 MB).</string>
```

Comment:
Same here, exceeds*

Does this behave as regular translation changes?
Where you have to update the .pot file as well?
I dont think so, but just making sure.

Reply:
You are right, I should also update the .po